packages/wagtail/modules/components/Blocks/RichTextBlock/index.tsx

```
@@ -0,0 +1,15 @@
+'use client'
+
+import { RichTextBlockWrapper } from './styled'
+import { RichTextBlockProps } from './types'
+
+const RichTextBlock = ({ value, WrapperProps }: RichTextBlockProps) => (
+  <RichTextBlockWrapper
+    {...WrapperProps}
+    // TODO (Tech Debt): Sanitize the HTML content with, for example, DOMPurify.
+    // eslint-disable-nex-line react/no-danger
+    dangerouslySetInnerHTML={{ __html: value }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)